### PR TITLE
Copy Admin Orders Code

### PR DESCRIPTION
**Pull request by lukecheseldine**
## Overview
- copied over code from the Admin Orders page

## Unit Tests Created

- none

## Steps to QA

- make sure i have all the required files, but the website runs as expected and tests pass so i think i do

### CODE REVIEW

File: hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx [hunk 1]
```diff
@@ -85,28 +95,13 @@ const Orders = () => {
                                         key={idx}
                                         onClick={() => handleClick(order.id)}
```
Comment (Mustaballer) [inline hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx]: Can you replace the onClick function with history.push(`/teams/${order.team_code}`)

File: hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx [hunk 2]
```diff
@@ -21,6 +29,8 @@ const Orders = () => {
         alert(`The order you clicked is: #${orderId}`);
```
Comment (Mustaballer) [inline hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx]: Can you remove this handleClick function, we don't need to send an alert. It should redirect admins to their team detail page. See my comment below, on how to do that.

File: hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx [hunk 3]
```diff
@@ -9,10 +9,18 @@ import OrdersFilter from "components/orders/OrdersFilter/OrderFilter";
...
+    adminOrderSelectors,
+    getOrdersWithFilters,
+} from "slices/order/adminOrderSlice";
 
 const Orders = () => {
+    const dispatch = useDispatch();
+    useEffect(() => {
+        dispatch(getOrdersWithFilters());
+    }, [dispatch]);
     const [mobileOpen, setMobileOpen] = React.useState(false);
```
Comment (Mustaballer) [inline hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx:25]: Since we are using historyobject below, add `const history = useHistory();`

**Review by Mustaballer — CHANGES_REQUESTED** [review]
Hey Luke great job! Just had a few comments and it will be good to merge :)

File: hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx (hunk 1 above)
Comment (lukecheseldine) [reply hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx]: done

File: hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx (hunk 2 above)
Comment (lukecheseldine) [reply hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx]: done

File: hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx (hunk 3 above)
Comment (lukecheseldine) [reply hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx:25]: done

**Review by Mustaballer — APPROVED** [review]
Good work!

**Pull request merged**